scio-parquet/src/main/scala/com/spotify/scio/parquet/GcsConnectorUtil.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.parquet
+
+import com.spotify.scio.ScioContext
+import com.spotify.scio.util.ScioUtil
+import org.apache.hadoop.mapreduce.Job
+import org.apache.hadoop.mapreduce.lib.input.FileInputFormat
+
+private[parquet] object GcsConnectorUtil {
+  def setCredentials(job: Job): Unit =
+    // These are needed since `FileInputFormat.setInputPaths` validates paths locally and
+    // requires the user's GCP credentials.
+    sys.env.get("GOOGLE_APPLICATION_CREDENTIALS") match {
```

Comment:
I see that this was just moved around, but would it be worth considering using `GoogleCredential` instead? This can be done in another PR. With this it would be consistent with the other IO's

Reply:
Not sure how you can use that. One branch requires the path to key JSON and the other just sets some hard coded key-values in Hadoop `Configuration`.